src/pages/SignInPage/SignInPage.jsx

```diff
@@ -53,10 +57,39 @@ const SignInPage = () => {
     }, 0);
   }, [pathname]);
 
-  const handleSubmit = (event) => {
+  // makes post petition to the backend
```

Comment:
// makes post request to the backend
But, thsi comment would be not necessary...